apps/frontend/src/extra_components/Carousel.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { useEffect, useMemo, useState } from "react";
+import {
+  Box,
+  IconButton,
+  Card,
+  Slide,
+  Typography,
+  Grid,
+  Paper,
+} from "@mui/material";
+import ArrowLeftIcon from "@mui/icons-material/ArrowLeft";
+import ArrowRightIcon from "@mui/icons-material/ArrowRight";
+
+export interface props {
+  imageURLs: string[];
+}
+const Carousel = ({ imageURLs }: props) => {
+  const [cards, setCards] = useState<React.ReactElement[]>([]);
+  const [currentPage, setCurrentPage] = useState(0);
+  const [slideDirection, setSlideDirection] = useState<
+    "right" | "left" | undefined
+  >("left");
+
+  const images: React.ReactElement[] = useMemo(() => {
+    return Array.from({ length: imageURLs.length }, (_, i) => <Card key={i} />);
+  }, [imageURLs]);
+
+  const handleNextPage = () => {
+    setSlideDirection("left");
```

Comment:
Since slideDirection is only ever left, it can probably just be hardcoded to left and the setSlideDirection logic can be removed